distribution/tools/plugin-cli/build.gradle

```diff
@@ -36,7 +36,7 @@ dependencies {
   compileOnly project(":server")
   compileOnly project(":libs:opensearch-cli")
   api "org.bouncycastle:bcpg-fips:1.0.4"
```

Comment:
There aren't any vulnerabilities for `bcpg-fips` but do you want to update it to 1.0.5.1 along with this change?

Reply:
Good catch 😄 I will give a refresh to it as well. Since they are used in conjunction, so it make sense to upgrade to a version released together.

Reply:
Now I gave a full refresh to all bouncy castle libraries. 🏰 Hope the tests pass.